src/AWS4AuthRequest.jl

```diff
@@ -60,6 +60,7 @@ function sign_aws4!(method::String,
                     aws_access_key_id::String=ENV["AWS_ACCESS_KEY_ID"],
                     aws_secret_access_key::String=ENV["AWS_SECRET_ACCESS_KEY"],
                     aws_session_token::String=get(ENV, "AWS_SESSION_TOKEN", ""),
+                    include_token_in_sig=true,
```

Comment:
Rename to `token_in_signature`? The `include_*` keywords below indicate if a header is included.